packages/grid/_modules_/grid/components/toolbar/GridToolbarFilterButton.tsx

```diff
@@ -95,7 +95,6 @@ export const GridToolbarFilterButton = React.forwardRef<HTMLButtonElement, Butto
       <Tooltip title={tooltipContentNode} enterDelay={1000}>
         <Button
           ref={ref}
-          {...props}
           onClick={toggleFilter}
```

Comment:
I don't think we want to be able to totally skip `toggleFilter`
It should be after the props spread.
And not directly related to this PR, the `onClick` should either be removed from the typing of the component or called in `toggleFilter`. Right now the typing allows to pass a `onClick` but does nothing with it.

Reply:
I guess I can exclude it from ButtonProps

Reply:
I can do the same for other toolbar component

Reply:
> I don't think we want to be able to totally skip toggleFilter

@flaviendelangle Agree, only with  #1403.